tensorboard/plugins/graph/tf_graph_op_compat_card/tf-graph-op-compat-card.ts

```diff
@@ -219,16 +217,12 @@ class TfGraphOpCompatCard extends LegacyElementMixin(PolymerElement) {
       list.fire('iron-resize');
     }
   }
-  @computed('graphHierarchy', 'hierarchyParams')
+  @computed('graphHierarchy')
   get _incompatibleOpNodes(): object {
```

Comment:
if you type the `getIncompatibleOps`, you can better type the return types.

Reply:
Good idea, done.